src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/ReplAction.scala

```diff
@@ -86,7 +86,7 @@ class ReplAction extends RunConsoleAction {
     //  virtual file is working for both: files and folders
     val targetFileOrFolder = CommonDataKeys.VIRTUAL_FILE.getData(dataContext)
 
-    if (project == null || checkFileOrFolderIsNull(targetFileOrFolder)) return
+    if (project == null || checkFileOrFolderIsNull(targetFileOrFolder)) return // scalastyle:ignore
```

Comment:
I think `return` statement is fine here. I guess these kind of `void`/`Unit` type methods are rarely used in (functional) Scala so need for `return`s does not occur often when coding pure Scala. As the design of this class is, however, based on its Java base class, it is natural that the code is structured Java/imperative way. Luckily, Scala supports this coding paradigm too (even though it might not be that "Scala-like")

Reply:
Maybe we need a third style checker that looks for ignore-comments :trollface: